sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala

```diff
@@ -125,8 +125,13 @@ object JdbcUtils extends Logging {
       dialect: JdbcDialect): Iterator[Byte] = {
     val conn = getConnection()
     var committed = false
+    var supportsTransactions = true
```

Comment:
Hi Sean,
in case a (flaky) driver throws some exception when detecting transaction support, the code should fall back to current standard behaviour: use transactions.
Does this make sense?
Thanks,
Christian